designs/administrator/4.6/guided-tours/index.md

```diff
@@ -0,0 +1,102 @@
+
+## Accessing tours
+Tours can be accessed on both the Administrator and Developer perspectives by going to the **Help menu** and selecting **Guided Tours**. Each of the perspectives will also surface links to the tours in other places around the console. For more detail around how these links will be surfaced in the Developer perspective, view the Developer side documentation.
+
+In the Administrator perspective, a badge will appear on the Overview page informing users that guided tours are available. When clicked, the badge will redirect users to the Guided Tours page.
+
+![1](img/admin1.png)
+
+The Guided Tours page will use a catalog-like design with each tour being shown on a tile. Tiles will provide a brief description of the tour, a time estimate for completing the tour, as well as prerequisites for completing the tour, if any.
+
+![2](img/admin2.png)
+
+If a user clicks on a tour tile, the tour side panel will open. The introductory panel will give users more information and allows the user to start the tour using the **Start tour** button.
+
+![3](img/tour1-screen0.png)
+
+## Tour permissions
+For 4.6, tour permissions will be as follows:
+* If a user can install Operators, show the first Serverless tour (Explore Serverless) which walks users through installing the Serverless Operator.
+* If a user cannot install Operators, do not show the first Serverless tour. Instead show an empty state. The empty state can inform users that tours will be added in future.
+* If an Operator has been installed, show the second Serverless tour (Serverless Applications) which walks users through deploying a Serverless application.
+* Any tour that has been completed should remain on the Guided Tours page with the option to **Review the tour** or **Restart the tour**.
+
+## Tour 1: Explore Serverless
+The first tour walks users through installing the Serverless Operator by completing a series of 3 tasks. Each task will have a number of sub steps that the user should execute in order to successfully complete the tour. Any links within the tour instructions will redirect users to that page in the console.
+
+### Check your work
+There will not be any validation from the console that users have successfully completed any sub steps or tasks. Instead, there will be an optional **Check your work** feature that asks users to validate that the tasks have been completed.
+
+![4](img/check1.png)
+
+The **Check your work** feature will appear after a user clicks **Next**. The user may click **Next** again to skip the validation step, or can read the instructions to make sure they are following the tour correctly.
+
+![5](img/check2.png)
```

Comment:
You nailed the reasoning behind this behavior :) We initially had it shown from the get-go but there were some concerns that it'd be too noisy and overwhelming. You raise a good question though. I guess I had imagined that it would go back to the previous step and not just remove the Check your work feature. Do you think that interaction makes sense?

Reply:
Going back to the previous step in one click makes sense to me, just wanted to verify. 👍

I don't have a strong opinion on the first click revealing Check your work and the second taking the user to the next step. I'll just wait to see how it feels in action. 🙂